### ISSUE TITLE
Support specifying alpha channel in colors

### ISSUE THREAD

**Issue by shanemhansen** [flxzt/rnote#62]
First of all, thanks for rnote! I spent the evening practicing my drawing skills with my wacom tablet. I was trying to highlight text and I noticed that while the marker tool has some transparent colors, I don't see a way to change them. If I select "custom" in the color picker I get what looks like a RGB color picker where I can only specify values like #RRGGBB rather than #RRGGBBAA

**Comment by flxzt** [comment]
Its the default Gtk colorpicker widget, while it doesn't display it as an additional hex value (which is not very intuitive, I agree), you can change the slider at the bottom to change the alpha. :)

**Comment by TornaxO7** [comment]
I've an idea for this:
![image](https://user-images.githubusercontent.com/50843046/230750333-d0d18efc-968c-4117-9750-782286bec6e8.png)

A third box which you can click to set the alpha-value either directly by a value or by a slider. Would that sound reasonable?

**Comment by flxzt** [comment]
I don't think this would be very intuitive. I would rather improve the color picker pop-up by implementing a fully custom one and add a box where the user can read / write `#rrggbbaa` values from / into it. By doing this we could also have some additional features like displaying a "recently used" palette by fetching the last colors from the current document

**Comment by TornaxO7** [comment]
well, that would be possible too. Just with a lot more work xD

**Comment by maymage** [comment]
Upon the risk of hijacking this issue... I think, in the colour selection or its submenu

* the standard colours or 

* black, white and transparent or 

* at least transparent 

should be unchangeably fixed *somewhere*, since 

* realistically no one needs more colours than the standard ones for note taking 

* today I had killed away transparent and couldn't get it (easily) back. 

Perhaps separating one of the former fixed cases from a few fields to choose colours to, would simplify things, kind of

```
|| line colour ||  fill colour || fixed colours black white transparent | preselected, changeable colours | submenu || 
```

**Comment by PatrickJosh** [comment]
If I may comment on that, I'd like to say that I would be against setting some immutable default colours, at least in the toolbar. I very much like the option to customise the colours that I use, even the ones that many people might consider default, such as black and white. For example, I prefer a black that is not `#000000` but maybe `#111111`.

**Comment by flxzt** [comment]
I agree with you. There is practically no downside to the ability to customize the palette, and the feature is already implemented. The comment above you should have been marked off-topic, it's not really what the original discussion was about
